pkgs/data/themes/gruvbox-dark-gtk/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "gruvbox-dark-gtk";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "jmattheis";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1wf4ybnjdp2kbbvz7pmqdnzk94axaqx5ws18f34hrg4y267n0w4g";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/themes/
+    cp -r ${src} $out/share/themes/${pname}
```

Comment:
In the distributed `index.theme` file, the `Name` field is set to `gruvbox-dark`, so it would be better to install to `$out/share/themes/gruvbox-dark`.

Maybe it is better not to install the `.github` subdirectory and the `LICENSE` and `README.md` files.